pkg/services/sqlstore/migrations/ualert/channel.go

```diff
@@ -114,7 +115,7 @@ func (m *migration) setupAlertmanagerConfigs(rulesPerOrg map[int64]map[*alertRul
 	return amConfigPerOrg, nil
 }
 
-// contactListToString creates a sorted string representation of a given map (set) of receiver names. Each name will be comma-separated and double-quoted. Double quotes inside a name are escaped.
+// contactListToString creates a sorted string representation of a given map (set) of receiver names. Each name will be comma-separated and double-quoted. Names should not contain double quotes.
 func contactListToString(m map[string]interface{}) string {
```

Comment:
Why is this restriction being relaxed?

(for my edification, not questioning the reason for the change)

Reply:
(I *think* it's because we already replace " with _)